packages/react-components/src/tooltip/src/TooltipTrigger.tsx

```diff
@@ -186,17 +194,17 @@ export function InnerTooltipTrigger({
                 {tooltipMarkup}
                 <OverlayArrow
                     style={arrowStyles}
-                    ref={setArrowElement}
+                    ref={setArrowElement as any}
```

Comment:
something feels really wrong here. setArrowElement is from a  useState function, not a useRef?